test/test_framework.py

```diff
@@ -473,7 +474,8 @@ def tearDown(self):
         shutil.rmtree(self.tmpdir)
 
     def create_framework(self):
-        return Framework(self.tmpdir / "framework.data")
+        framework = Framework(self.tmpdir / "framework.data")
+        return framework
```

Comment:
Oops.  Another pointless change that got left in from some interactive test debugging.